js/src/features/human-resouce/selectors.ts

```diff
@@ -29,3 +29,6 @@ export const isAssignLocationFormVisible = (state: RootState) =>
 
 export const isEditEmployeeLocationFormVisible = (state: RootState) =>
   state.humanResource.location.employeeLocationToEdit !== null;
+
+export const selectVacations = (state: RootState) =>
```

Comment:
Why `selectVacations` ? I remember you proposed to name selectors with `get`. Maybe `getVacations` ?

Reply:
I was a little wrong then. In redux documentation, variables start with select.

Reply:
ok